runtime/Cargo.toml

```diff
@@ -144,6 +175,13 @@ git = 'https://github.com/paritytech/substrate.git'
 version = '4.0.0-dev'
 branch = "master"
 
+## This might be needed during voting.
```

Comment:
Is this still unsure or can this code be removed?

Reply:
removed